libparsec/crates/platform_device_loader/tests/archive.rs

```diff
@@ -3,6 +3,7 @@
 // `allow-unwrap-in-test` don't behave as expected, see:
 // https://github.com/rust-lang/rust-clippy/issues/11119
 #![allow(clippy::unwrap_used)]
+#![cfg(not(target_arch = "wasm32"))]
```

Comment:
Should add a `// TODO: Web support is not implemented yet` comment on those conditionals

Reply:
done